VoxelSniperForge/src/main/java/com/github/kevindagame/voxelsniperforge/blockstate/sign/ForgeSign.java

```diff
@@ -18,18 +23,35 @@ public ForgeSign(ForgeBlock block, BlockState blockState, SignBlockEntity tileEn
         this.tileEntity = tileEntity;
     }
 
-    @NotNull
-    private static String translateColorCodesToAmpersand(@NotNull String textToTranslate) {
-        throw new UnsupportedOperationException("Not implemented yet");
+    private SignText front() {
+        return tileEntity.getFrontText();
+    }
+
+    private void front(SignText text) {
+        boolean front = true;
+        tileEntity.setText(text, front);
```

Comment:
This method needs another name, or some documentation, also, boolean front can just be inlined

Reply:
I inlined both functions, I like the separate boolean because Forge doesn't tell you what the parameter does...